AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts

```diff
@@ -29,6 +30,7 @@ export class ParticipantModel {
     is_staff_member?: boolean;
     contact_email?: string;
     isJudiciaryMember?: boolean;
+    interpretationLanguage: InterpreterSelectedDto;
```

Comment:
we have a mix of snake_case and camelCase conventions for these property names, perhaps we should stick with one or the other. Maybe snake_case as that is the majority?

Reply:
would be nice to switch to camelCase as the rest of the repo but we'll save that for another day